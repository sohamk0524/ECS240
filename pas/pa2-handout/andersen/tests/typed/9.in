c Test case with mix of all statements
p 205 320
t 1 40
t 2 40
t 3 40
t 4 40
t 5 40
t 6 39
t 7 39
t 8 39
t 9 39
t 10 39
t 11 38
t 12 38
t 13 38
t 14 38
t 15 38
t 16 37
t 17 37
t 18 37
t 19 37
t 20 37
t 21 36
t 22 36
t 23 36
t 24 36
t 25 36
t 26 35
t 27 35
t 28 35
t 29 35
t 30 35
t 31 34
t 32 34
t 33 34
t 34 34
t 35 34
t 36 33
t 37 33
t 38 33
t 39 33
t 40 33
t 41 32
t 42 32
t 43 32
t 44 32
t 45 32
t 46 31
t 47 31
t 48 31
t 49 31
t 50 31
t 51 30
t 52 30
t 53 30
t 54 30
t 55 30
t 56 29
t 57 29
t 58 29
t 59 29
t 60 29
t 61 28
t 62 28
t 63 28
t 64 28
t 65 28
t 66 27
t 67 27
t 68 27
t 69 27
t 70 27
t 71 26
t 72 26
t 73 26
t 74 26
t 75 26
t 76 25
t 77 25
t 78 25
t 79 25
t 80 25
t 81 24
t 82 24
t 83 24
t 84 24
t 85 24
t 86 23
t 87 23
t 88 23
t 89 23
t 90 23
t 91 22
t 92 22
t 93 22
t 94 22
t 95 22
t 96 21
t 97 21
t 98 21
t 99 21
t 100 21
t 101 20
t 102 20
t 103 20
t 104 20
t 105 20
t 106 19
t 107 19
t 108 19
t 109 19
t 110 19
t 111 18
t 112 18
t 113 18
t 114 18
t 115 18
t 116 17
t 117 17
t 118 17
t 119 17
t 120 17
t 121 16
t 122 16
t 123 16
t 124 16
t 125 16
t 126 15
t 127 15
t 128 15
t 129 15
t 130 15
t 131 14
t 132 14
t 133 14
t 134 14
t 135 14
t 136 13
t 137 13
t 138 13
t 139 13
t 140 13
t 141 12
t 142 12
t 143 12
t 144 12
t 145 12
t 146 11
t 147 11
t 148 11
t 149 11
t 150 11
t 151 10
t 152 10
t 153 10
t 154 10
t 155 10
t 156 9
t 157 9
t 158 9
t 159 9
t 160 9
t 161 8
t 162 8
t 163 8
t 164 8
t 165 8
t 166 7
t 167 7
t 168 7
t 169 7
t 170 7
t 171 6
t 172 6
t 173 6
t 174 6
t 175 6
t 176 5
t 177 5
t 178 5
t 179 5
t 180 5
t 181 4
t 182 4
t 183 4
t 184 4
t 185 4
t 186 3
t 187 3
t 188 3
t 189 3
t 190 3
t 191 2
t 192 2
t 193 2
t 194 2
t 195 2
t 196 1
t 197 1
t 198 1
t 199 1
t 200 1
t 201 0
t 202 0
t 203 0
t 204 0
t 205 0
s 0 1 -1 6
s 0 5 -1 9
s 0 3 -1 10
s 0 5 0 3
s 0 1 0 5
s 0 3 0 1
s 0 1 0 3
s 0 5 0 1
s 0 6 -1 14
s 0 10 -1 12
s 0 7 -1 12
s 1 3 0 7
s 0 7 0 10
s 1 3 0 6
s 1 1 1 5
s 1 3 1 5
s 0 11 -1 16
s 0 15 -1 20
s 0 12 -1 17
s 2 1 1 7
s 0 15 0 11
s 0 15 2 1
s 1 6 1 10
s 2 3 0 12
s 0 16 -1 24
s 0 20 -1 25
s 0 17 -1 21
s 0 17 2 10
s 0 17 0 20
s 1 12 2 6
s 1 15 2 10
s 3 5 2 6
s 0 21 -1 27
s 0 25 -1 30
s 0 22 -1 28
s 2 12 2 11
s 0 25 2 11
s 1 17 0 25
s 1 16 3 10
s 4 5 0 25
s 0 26 -1 32
s 0 30 -1 35
s 0 29 -1 35
s 4 7 2 16
s 1 21 2 17
s 2 16 5 5
s 1 25 3 12
s 0 30 3 11
s 0 31 -1 37
s 0 35 -1 37
s 0 34 -1 38
s 2 21 5 6
s 2 22 0 34
s 1 26 5 6
s 6 1 5 10
s 2 21 3 16
s 0 36 -1 41
s 0 40 -1 44
s 0 37 -1 42
s 1 31 0 36
s 7 1 7 3
s 2 26 0 36
s 5 12 1 34
s 4 17 5 12
s 0 41 -1 50
s 0 45 -1 48
s 0 43 -1 47
s 5 17 2 35
s 0 43 4 21
s 3 29 6 12
s 0 41 0 45
s 3 29 3 26
s 0 46 -1 53
s 0 50 -1 53
s 0 48 -1 53
s 6 16 8 10
s 8 6 6 20
s 9 3 3 31
s 3 31 2 40
s 0 50 3 31
s 0 51 -1 56
s 0 55 -1 60
s 0 52 -1 60
s 4 31 7 16
s 4 31 4 31
s 0 52 2 41
s 8 15 7 17
s 9 10 0 52
s 0 56 -1 63
s 0 60 -1 62
s 0 59 -1 65
s 6 26 9 11
s 0 59 0 60
s 2 50 0 59
s 5 31 3 41
s 11 3 4 36
s 0 61 -1 66
s 0 65 -1 69
s 0 63 -1 67
s 7 26 2 51
s 10 12 3 50
s 7 26 2 55
s 8 25 9 16
s 1 59 0 61
s 0 66 -1 75
s 0 70 -1 74
s 0 68 -1 71
s 6 40 0 68
s 5 45 13 3
s 11 11 7 35
s 13 1 0 70
s 10 17 6 36
s 0 71 -1 80
s 0 75 -1 79
s 0 74 -1 77
s 11 20 3 56
s 9 30 1 70
s 12 15 2 61
s 6 45 14 3
s 10 25 12 15
s 0 76 -1 85
s 0 80 -1 82
s 0 78 -1 82
s 15 5 15 1
s 5 52 14 10
s 9 34 3 61
s 15 1 7 45
s 5 52 6 48
s 0 81 -1 87
s 0 85 -1 89
s 0 83 -1 88
s 0 81 0 83
s 0 81 7 48
s 0 85 5 59
s 11 30 2 74
s 3 70 13 16
s 0 86 -1 94
s 0 90 -1 92
s 0 87 -1 95
s 3 75 16 6
s 1 81 3 71
s 9 43 14 20
s 15 11 11 35
s 1 85 4 68
s 0 91 -1 97
s 0 95 -1 100
s 0 94 -1 96
s 8 55 9 50
s 7 56 10 43
s 10 41 5 68
s 13 26 2 85
s 4 74 16 11
s 0 96 -1 103
s 0 100 -1 103
s 0 98 -1 105
s 1 95 2 90
s 12 40 8 60
s 3 85 13 34
s 7 61 9 55
s 4 76 19 1
s 0 101 -1 109
s 0 105 -1 107
s 0 104 -1 108
s 5 80 0 105
s 1 100 10 52
s 11 46 7 70
s 5 78 8 63
s 17 20 12 41
s 0 106 -1 111
s 0 110 -1 114
s 0 109 -1 111
s 7 74 1 105
s 3 91 11 52
s 20 10 4 90
s 18 20 0 110
s 14 36 8 66
s 0 111 -1 120
s 0 115 -1 116
s 0 113 -1 119
s 12 52 22 3
s 9 66 11 60
s 18 25 14 45
s 2 101 13 50
s 4 95 15 40
s 0 116 -1 122
s 0 120 -1 123
s 0 118 -1 123
s 11 61 18 30
s 23 3 0 120
s 23 1 17 35
s 10 66 5 91
s 8 78 2 109
s 0 121 -1 129
s 0 125 -1 128
s 0 124 -1 128
s 19 30 6 95
s 7 86 24 5
s 21 16 22 12
s 10 71 7 87
s 11 68 3 110
s 0 126 -1 135
s 0 130 -1 134
s 0 128 -1 135
s 18 40 0 126
s 8 90 23 12
s 3 111 0 126
s 5 101 17 43
s 10 76 12 70
s 0 131 -1 139
s 0 135 -1 140
s 0 132 -1 136
s 7 98 18 43
s 11 80 26 3
s 12 74 14 63
s 12 74 26 1
s 20 34 25 10
s 0 136 -1 142
s 0 140 -1 144
s 0 138 -1 141
s 4 118 0 136
s 23 21 20 36
s 14 70 24 16
s 10 87 0 140
s 2 128 24 17
s 0 141 -1 146
s 0 145 -1 146
s 0 144 -1 146
s 10 94 22 34
s 27 10 6 111
s 15 70 24 21
s 20 43 22 35
s 11 86 26 11
s 0 146 -1 153
s 0 150 -1 151
s 0 149 -1 152
s 20 46 11 95
s 16 70 26 20
s 17 63 23 34
s 11 95 6 116
s 6 116 22 40
s 0 151 -1 159
s 0 155 -1 160
s 0 153 -1 158
s 4 135 18 65
s 16 74 2 141
s 30 5 6 121
s 28 15 23 36
s 1 149 29 6
s 0 156 -1 161
s 0 160 -1 163
s 0 157 -1 162
s 3 141 23 45
s 9 115 24 37
s 23 43 24 36
s 4 136 0 157
s 28 16 4 140
s 0 161 -1 170
s 0 165 -1 169
s 0 162 -1 169
s 30 11 30 15
s 1 160 18 75
s 10 111 31 6
s 19 70 28 21
s 0 162 20 63
s 0 166 -1 172
s 0 170 -1 173
s 0 168 -1 171
s 2 160 9 121
s 0 168 21 63
s 23 51 19 75
s 9 121 15 94
s 26 40 8 128
s 0 171 -1 176
s 0 175 -1 176
s 0 173 -1 180
s 10 121 20 75
s 3 160 6 144
s 23 59 2 162
s 14 104 3 160
s 0 175 17 90
s 0 176 -1 181
s 0 180 -1 183
s 0 177 -1 183
s 23 65 27 43
s 15 104 4 160
s 7 141 32 17
s 22 70 6 150
s 33 11 16 98
s 0 181 -1 187
s 0 185 -1 187
s 0 184 -1 190
s 15 106 17 96
s 26 51 20 83
s 4 162 20 85
s 9 140 16 101
s 34 15 34 15
s 0 186 -1 194
s 0 190 -1 193
s 0 189 -1 194
s 30 40 27 55
s 32 26 37 5
s 3 173 12 128
s 24 66 23 74
s 33 21 8 146
s 0 191 -1 198
s 0 195 -1 200
s 0 194 -1 200
s 25 68 24 75
s 27 60 24 71
s 26 65 20 94
s 0 194 22 81
s 37 7 12 131
s 0 196 -1 203
s 0 200 -1 203
s 0 197 -1 201
s 16 120 33 35
s 2 190 6 170
s 4 177 26 70
s 19 101 23 83
s 28 59 27 63
