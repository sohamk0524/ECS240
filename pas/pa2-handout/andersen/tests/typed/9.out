pt 100 103
pt 101 109
pt 10 12
pt 10 14
pt 104 108
pt 105 107
pt 106 111
pt 109 111
pt 1 10
pt 110 114
pt 111 120
pt 11 16
pt 113 119
pt 115 116
pt 116 122
pt 118 123
pt 120 123
pt 121 129
pt 12 17
pt 124 128
pt 125 128
pt 126 135
pt 128 135
pt 130 134
pt 131 139
pt 132 136
pt 135 140
pt 136 142
pt 138 141
pt 140 144
pt 141 146
pt 14 17
pt 144 146
pt 145 146
pt 146 153
pt 149 152
pt 150 151
pt 151 159
pt 15 16
pt 15 17
pt 15 20
pt 153 158
pt 155 160
pt 156 161
pt 157 162
pt 1 6
pt 160 163
pt 161 170
pt 16 21
pt 162 169
pt 16 24
pt 16 25
pt 165 169
pt 166 172
pt 168 171
pt 170 173
pt 171 176
pt 17 21
pt 17 25
pt 173 180
pt 175 176
pt 176 181
pt 177 183
pt 180 183
pt 181 187
pt 184 190
pt 185 187
pt 186 194
pt 189 194
pt 1 9
pt 190 193
pt 191 198
pt 194 200
pt 195 200
pt 196 203
pt 197 201
pt 200 203
pt 20 21
pt 20 25
pt 21 27
pt 21 30
pt 22 28
pt 24 27
pt 24 30
pt 25 27
pt 25 30
pt 26 32
pt 27 35
pt 29 35
pt 30 35
pt 3 10
pt 31 37
pt 32 37
pt 34 38
pt 35 37
pt 3 6
pt 36 41
pt 37 41
pt 37 42
pt 3 9
pt 40 44
pt 41 48
pt 41 50
pt 43 47
pt 43 48
pt 43 50
pt 45 48
pt 46 53
pt 48 53
pt 50 53
pt 5 10
pt 51 56
pt 52 60
pt 53 60
pt 55 60
pt 5 6
pt 56 63
pt 5 9
pt 59 62
pt 59 65
pt 60 62
pt 60 65
pt 6 12
pt 6 14
pt 61 66
pt 62 66
pt 62 67
pt 63 67
pt 65 66
pt 65 67
pt 65 69
pt 66 74
pt 66 75
pt 67 74
pt 68 71
pt 69 74
pt 70 74
pt 7 12
pt 7 14
pt 71 80
pt 74 77
pt 74 79
pt 75 77
pt 75 79
pt 76 85
pt 78 82
pt 80 82
pt 81 87
pt 81 88
pt 83 88
pt 85 89
pt 86 94
pt 87 95
pt 90 92
pt 9 12
pt 9 14
pt 91 97
pt 94 96
pt 95 100
pt 96 103
pt 98 105
